onair/src/run_scripts/execution_engine.py

```diff
@@ -56,59 +57,77 @@ def __init__(self, config_file='', run_name='', save_flag=False):
-            self.fullTelemetryFileName = os.path.join(self.dataFilePath, self.telemetryFile)
-            self.metadataFilePath = config['DEFAULT']['TelemetryMetadataFilePath']
-            self.metaFile = config['DEFAULT']['MetaFile'] # Config for vehicle telemetry
-            self.fullMetaDataFileName = os.path.join(self.metadataFilePath, self.metaFile)
-
-            ## Parse Required Data: Names
-            self.parser_file_name = config['DEFAULT']['ParserFileName']
-
-            ## Parse Required Data: Plugins
-            self.knowledge_rep_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['KnowledgeRepPluginDict'])
-            self.learners_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['LearnersPluginDict'])
-            self.planners_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['PlannersPluginDict'])
-            self.complex_plugin_dict = self.parse_plugins_dict(config['DEFAULT']['ComplexPluginDict'])
-
-            ## Parse Optional Data: Flags
-            ## 'RUN_FLAGS' must exist, but individual flags return False if missing
-            self.IO_Flag = config['RUN_FLAGS'].getboolean('IO_Flag')
+            # Parse Required Data: FILES
+            self.dataFilePath = config['FILES']['TelemetryFilePath']
+            # Vehicle telemetry data
+            self.telemetryFile = config['FILES']['TelemetryFile']
+            self.fullTelemetryFile = os.path.join(
+                self.dataFilePath, self.telemetryFile)
+            self.metadataFilePath = config['FILES']['MetaFilePath']
+            # Config for vehicle telemetry
+            self.metaFile = config['FILES']['MetaFile']
+            self.fullMetaFile = os.path.join(
+                self.metadataFilePath, self.metaFile)
+
+            # Parse Required Data: DATA_HANDLING
+            self.data_source_file = config['DATA_HANDLING']['DataSourceFile']
+
+            # Parse Required Data: PLUGINS
+            self.knowledge_rep_plugin_dict = self.parse_plugins_dict(
+                config['PLUGINS']['KnowledgeRepPluginDict'])
+            self.learners_plugin_dict = self.parse_plugins_dict(
+                config['PLUGINS']['LearnersPluginDict'])
+            self.planners_plugin_dict = self.parse_plugins_dict(
+                config['PLUGINS']['PlannersPluginDict'])
+            self.complex_plugin_dict = self.parse_plugins_dict(
+                config['PLUGINS']['ComplexPluginDict'])
+
+            # Parse Optional Data: OPTIONS
+            # 'OPTIONS' must exist, but individual options return False if missing
+            if config.has_section('OPTIONS'):
+                self.IO_Enabled = config['OPTIONS'].getboolean('IO_Enabled')
+            else:
+                self.IO_Enabled = False
 
         except KeyError as e:
-            new_message = f"Config file: '{config_filepath}', missing key: {e.args[0]}"
+            new_message = f"Config file: '{
+                config_filepath}', missing key: {e.args[0]}"
```

Comment:
f-string does not allow splitting lines < Python 3.12
We are currently supporting Python 3.8-3.12. 
Suggest we allow the long lines or we'll need to end strings with \ (as well as not using the / within the braces {}) AND begin the next line left justified.
```
        new_message = f"Config file: '\
{config_filepath}', missing key: {e.args[0]}"
```